Cargo.toml

```diff
@@ -44,7 +44,11 @@ sha2 = { version = "0.8", optional = true }
 x25519-dalek = { version = "0.5", optional = true }
 
 # ring crypto proivder
-ring = { version = "0.15", optional = true }
+[dependencies.ring]
+version = "0.16"
+optional = true
+default-features = true
```

Comment:
is this needed? I'd personally prefer the shorter one-liner dependency a la:

```toml
ring = { version = "^0.16.2", optional = true, features = ["std"] }
```

when possible.